.github/ISSUE_TEMPLATE/docs-request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Documentation Request
+about: General request for the Observability Documentation Team.
+title: "[Request]"
+labels: Team:Docs
+assignees: ''
+projects: 513
+
+---
+
+## Description
+*(Describe what needs to be documented. Link to relevant issues welcome)*
+
+## Collaboration
+
+* [ ] The docs team will lead producing the content
+* [ ] The product team will provide the initial content and the docs team will edit / review.
+* [ ] The docs team will define with product team the structure and location and the product team will provide the initial content
+* [ ] Other (please describe)
+
+## Suggested Target Date
+*(let us know if this feature is needed for a specific release)*
```

Comment:
```suggestion
*(Let us know if this feature is needed for a specific release.)*
```

Reply:
I liked your idea in the weekly to change the title away from target date to target version/release